src/components/Events.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from "react";
+
+export const mockEvents = [
+  {
+    id: 1,
+    name: "Vendy",
+    sport: "Basketball",
+  },
+  {
+    id: 2,
+    name: "Lukas",
+    sport: "Football",
+  },
+  {
+    id: 3,
+    name: "Honza",
+    sport: "Tennis",
+  },
+  {
+    id: 4,
+    name: "Vendy",
+    sport: "Box",
+  },
+  {
+    id: 5,
+    name: "Lukas",
+    sport: "Skydiving",
+  },
+  {
+    id: 6,
+    name: "Johana",
+    sport: "Jogging",
+  },
+];
+
+export const Events: FC = () => {
+  return (
```

Comment:
I will exceptionally comment on this. It is a mistake on my side, that the eslint/prettier is not yet set up.

For codebases that do prefer ES6 (es2015) which should be any modern codebase. It is a good practice to strive toward a so-called "implicit" return. If you are returning 1 expression from the es6 function (arrow function) then you can get rid of the "return" word and the curly braces

So it would be 

```
export const Events: FC = () =>  (
    <>
      {mockEvents.map((event) =>
... rest of the code
)
```

Reply:
cool!